### PR TITLE
Update teams-exploratory.md

### PR DESCRIPTION
**Pull request by atvats**
Added the link to the TeamsBlockedTenantIds.csv file.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-exploratory.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Users must be enabled to sign up for apps and trials (in the Microsoft 365 admin
 Users don't fit the criteria if they:
 
 - Currently or previously had Teams from a paid, unpaid, or trial license
-- Are in a tenant that used/received at least one special COVID offer.
+- Are in a tenant that used/received at least one special COVID offer.This is the link to the list of tenantIds currently blocked by the signup service : https://dev.azure.com/microsoft/Universal%20Store/_git/BSX.Signup.Aloha/pullrequest/5282454?_a=files&path=%2Fsrc%2FCommon%2FDomainsBusinessLayer%2FDomainLists%2FTeamsBlockedTenantIds.csv
```
Comment (JohanFreelancer9) [inline Teams/teams-exploratory.md:66]:
```suggestion
- Are in a tenant that used/received at least one special COVID offer. The following link lists the tenant IDs currently blocked by the signup service: https://dev.azure.com/microsoft/Universal%20Store/_git/BSX.Signup.Aloha/pullrequest/5282454?_a=files&path=%2Fsrc%2FCommon%2FDomainsBusinessLayer%2FDomainLists%2FTeamsBlockedTenantIds.csv
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by SerdarSoysal** [comment]
This is an internal resource and has no place in public facing documentation. Rejected.

**Pull request closed without merging**